.github/kokoro/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+SCRIPT_SRC="$(realpath ${BASH_SOURCE[0]})"
+SCRIPT_DIR="$(dirname "${SCRIPT_SRC}")"
+INSTALL_DIR="$(pwd)/install"
+GIT_DESCRIBE=$(git describe)
+
+export CMAKE_FLAGS="-GNinja -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}"
```

Comment:
Please use the explicit A200T disable option here

Reply:
If we are going to perform the install step in a separate ci build, IMO we could avoid having the disable/enable options and just run the make install for all the supported architectures.

Disabling the a200t was necessary as previously, make install was running on the xc7 target.

Reply:
Ok